src/components/content/resource/resource-table-actions.tsx

```diff
@@ -30,16 +30,29 @@ export const ResourceTableActions = <T extends MinRecordItem>({
     <div
       className={cx(
         className,
-        "ctw-flex ctw-flex-wrap ctw-items-center ctw-justify-between ctw-py-2 sm:ctw-pt-1.5"
+        "ctw-flex ctw-items-start ctw-justify-between ctw-pt-2 sm:ctw-pt-1.5"
       )}
     >
-      <div className="ctw-flex ctw-flex-wrap ctw-gap-x-2">
-        {viewOptions && <ViewButton {...viewOptions} />}
-        {sortOptions && <SortButton {...sortOptions} />}
+      <div className="ctw-flex ctw-flex-col ctw-flex-wrap ctw-gap-x-2 md:ctw-flex-row">
+        {viewOptions && (
+          <ViewButton
+            className={cx(viewOptions.className, "ctw-mb-2")}
```

Comment:
because of the `ctw-gap-2` from the parent, you can remove the places where you used `ctw-mb-2`! :D

```suggestion
            className={viewOptions.className}
```